pkg/cloudprovider/aws/amifamily/bottlerocket.go

```diff
@@ -94,3 +94,11 @@ func (b Bottlerocket) ENILimitedMemoryOverhead() bool {
 func (b Bottlerocket) PodsPerCoreEnabled() bool {
 	return false
 }
+
+// EvictionSoftEnabled is currently disabled for Bottlerocket AMIFamily because it does
+// not currently support the evictionSoft parameter passed through the kubernetes settings TOML userData
+// If a Provisioner sets the evictionSoft value when using the Bottlerocket AMIFamily in the provider,
+// evictionSoft will be ignored
```

Comment:
can you link any applicable issues in BR or create one in this comment so we can check on it.